cleanup-evicted-pods/README.md

```diff
@@ -0,0 +1,16 @@
+# Cleanup evicted pods left behind after disk pressure
+When a node starts to evict pods under disk pressure, the evicted pods are left behind. All the resources like volumes, IP, containers, etc will be cleaned up and delete. But the pod object will be left behind in "evicted" status. Per upstream this is [intentional](https://github.com/kubernetes/kubernetes/issues/54525#issuecomment-340035375)
+
+## Workaround
+
+### Manual cleanup
+```bash
+kubectl get pods --all-namespaces -ojson | jq -r '.items[] | select(.status.reason!=null) | select(.status.reason | contains("Evicted")) | .metadata.name + " " + .metadata.namespace' | xargs -n2 -l bash -c 'kubectl delete pods $0 --namespace=$1'
```

Comment:
Do we need `select(.status.reason!=null)` ?

Reply:
I get the following error if that is removed.

```
> kubectl get pods --all-namespaces -ojson | jq -r '.items[] | select(.status.reason | contains("Evicted")) | .metadata.name + " " + .metadata.namespace'
jq: error (at <stdin>:13385): null (null) and string ("Evicted") cannot have their containment checked
> 
```

Reply:
Thanks, Looks like jq can't loop through null/noexisting objects.